xlgui/collection.py

```diff
@@ -186,7 +181,7 @@ def on_force_rescan_button_clicked(self, widget):
         from xlgui import main
         main.mainwindow().controller.on_rescan_collection_forced()
         
-    
+    @GtkTemplate.Callback
```

Comment:
`GtkTemplateWarning: Signal 'on_selection_changed' was declared with @GtkTemplate.Callback but was not present in template`